charts/mlp/tests/mlp_cm_tests.yaml

```diff
@@ -0,0 +1,71 @@
+suite: Unit tests for mlp config map values
+templates:
+  - configmap.yaml
```

Comment:
are we going to use unit tests for each template individually? if yes, should we name tests accordingly?

Reply:
Not necessarily, We can have multiple templates in the same suite and add another key inside each test indicating which template it needs if needed. 

I started this config map test file thinking it would have enough scenarios to make it a considerably large file. Hence the suite and the file are named accordingly. Once I'm done with the scenarios I will see if multiple files can be combined. 


>to use unit tests for each template individually
if you mean about invoking them, We will execute all the tests in one go for every chart
`helm unittest --helm3 -u -f tests/* . --debug`.